### ISSUE TITLE
Image containing manual labels is not cropped

### ISSUE THREAD

**Issue by PaulBautin** [sct-pipeline/csa-atrophy#66]
Image containing manual labels is not cropped. Segmentation and labeling images do not have the same space. This is probably causing the occuring errors in sct_process_segmentation with commit https://github.com/sct-pipeline/csa-atrophy/commit/ee367a4ef4339b90c7d29a6e160cfd26534e5c96

TODO:
- Add cropping to image containing manual labels.

**Comment by jcohenadad** [comment]
@PaulBautin i can help you with this-- can you please upload in this issue's body
- some "toy data" 
- the current syntax

so i can reproduce the issue, and find a fix for it

**Comment by PaulBautin** [comment]
This file needs cropping:
[sub-balgrist05_T1w_RPI_r_crop_r1_seg_labeled_t9.zip](https://github.com/sct-pipeline/csa-atrophy/files/5157321/sub-balgrist05_T1w_RPI_r_crop_r1_seg_labeled_t9.zip)

Up to now we are using:
```
# dilate segmentation (for cropping)
sct_maths -i ${file_seg}.nii.gz -dilate 15 -shape cube -o ${file_seg}_dil.nii.gz
# crop image
sct_crop_image -i ${file}.nii.gz -m ${file_seg}_dil.nii.gz
file=${file}_crop
# crop segmentation
sct_crop_image -i ${file_seg}.nii.gz -m ${file_seg}_dil.nii.gz
file_seg=${file_seg}_crop
```
I think adding the same sct_crop_image lines for the labeling should work.

**Comment by jcohenadad** [comment]
yup, that makes sense.
but i'm wondering why not all images crashed, then?

**Comment by PaulBautin** [comment]
And why it is called crop when it is not cropped at all??

**Comment by jcohenadad** [comment]
> And why it is called crop when it is not cropped at all??

ah yeah... good point.  If you touch that code, i would suggest to slightly change the naming:
`sub-balgrist05_T1w_RPI_r_crop_r1_seg_labeled_t9.nii.gz` --> `sub-balgrist05_T1w_RPI_r_crop_r1_t9_seg_labeled.nii.gz` (so to be consistend with the image and the image_seg)

**Comment by PaulBautin** [comment]
> If you touch that code, i would suggest to slightly change the naming:
> `sub-balgrist05_T1w_RPI_r_crop_r1_seg_labeled_t9.nii.gz` --> `sub-balgrist05_T1w_RPI_r_crop_r1_t9_seg_labeled.nii.gz` (so to be consistend with the image and the image_seg)

@jcohenadad, i did not yet implement your suggestion ^ in PR #67. I think the easiest convention is to add suffixes with each process in pipeline (easier to keep track of process). 



> And why it is called crop when it is not cropped at all??

This ^ was an issue because it did not follow the convention.

**Comment by PaulBautin** [comment]
Closing issue: The PRs associated with this issue, PR #67 (closed without merge) and PR #70 were closed or merged.
